app/views/user_mailer/account_reset_granted.html.erb

```diff
@@ -47,7 +47,7 @@
 </table>
 
 <p>
-  <%= t('user_mailer.email_confirmation_instructions.footer', confirmation_period: '24 hours') %>
+  <%= t('user_mailer.email_confirmation_instructions.footer', confirmation_period: @confirmation_period) %>
```

Comment:
`@confirmation_period` doesn't appear to be set to anything, this is blank in the email:

![image](https://github.com/18F/identity-idp/assets/1779930/8c896adc-3878-413e-b469-ed5c33cac9c3)

http://localhost:3000/rails/mailers/user_mailer/account_reset_granted

Reply:
Thanks! Fixed this in the following commit